sam-karis/airtech
# # add locust and Heroku configuration

## What does this PR do?
------
It adds locust configuration for monitoring the application and configuration for Heroku deployment

## Description of Task to be completed?
-----
- add locust configurations
- add Heroku configurations
- update ticket and flight models


## How should this be manually tested?
------
clone the repo
check out to `ft-locust-167660084`
add environment variables as defined in [`.env_example`](https://github.com/sam-karis/airtech/blob/develop/.env_example)
Create a database and run `python manage.py migrate` to create relevant tables.


## Any background context you want to provide?
------
None

## What are the relevant pivotal tracker stories?
------
[locust and Heroku config](https://www.pivotaltracker.com/story/show/167660084)


## Screenshots (if appropriate)
------
#### locust screenshot
![image](https://user-images.githubusercontent.com/22454909/63623898-88924680-c603-11e9-8907-d9ecaf134cf2.png)
